source/includes/dashboard/_90-get-dashboard.md.erb

```diff
@@ -107,11 +113,19 @@ Content-Type: <%= data.apis.versions.dashboard %>; charset=utf-8
 
 Lists pipelines on Dashboard along with pipeline groups and environments information.
 
+You can pass a personalized view name as a query string to the url to get specific pipelines.
```

Comment:
`optionally`?

You can **optionally** pass a personalized view name as a query string, to get all pipelines from the specified view.